Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
Yes I understand. Most package managers do this though ([example](https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-openjpeg/PKGBUILD)), it is the only way to get cmake to build both the dynamic and static library. Note that openjpeg is super small, the full build process takes about 2sec, so performance shouldn't be too much of an issue in this case I think.